hub/powertoys/grouppolicy.md

```diff
@@ -23,48 +27,161 @@ Since version 0.64, PowerToys is released on GitHub with [Administrative Templat
 2. Copy the "PowerToys.admx" file to the PolicyDefinition folder. (Example: %systemroot%\sysvol\domain\policies\PolicyDefinitions)
 3. Copy the "PowerToys.adml" file to the matching language folder in the PolicyDefinition folder. Create the folder if it doesn't already exist. (Example: %systemroot%\sysvol\domain\policies\PolicyDefinitions\EN-US)
 4. If your domain has more than one domain controller, the new ADMX files will be replicated to them at the next domain replication interval.
+### Import the administrative template in Intune
+
+You can find all instructions on how to import the tdministrative templates in Intune [here](https://learn.microsoft.com/mem/intune/configuration/administrative-templates-import-custom#add-the-admx-and-adml-files).
 
 ### Scope
 
 You will find the policies under "Administrative Templates/Microsoft PowerToys" in both the Computer Configuration and User Configuration folders. If both settings are configured, the setting in Computer Configuration takes precedence over the setting in User Configuration.
 
 ## Policies
 
+<!--Note for every dev who updates this file:
+The syntax of OMA-URI is the following: ./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys[[~<category1>]~<categoryN>]/<ADMX-DisplayName-ID>
+-->
+
 ### Configure global utility enabled state
 
-> [!NOTE]
-> This policy is supported on PowerToys 0.75.0 and later.
+> Supported on PowerToys 0.75.0 or later.
 
 This policy configures the enabled state for all PowerToys utilities.
 
 - If you enable this setting, all utilities will be always enabled and the user won't be able to disable it.
 - If you disable this setting, all utilities will be always disabled and the user won't be able to enable it.
-- If you don't configure this setting, users are able to disable or enable the utilities.
+- If you don't configure this setting, users are able to enable or disable the utilities.
 
 The individual enabled state policies for the utilities will override this policy.
 
+#### Group Policy (ADMX) info
+
+- GP unique name: ConfigureGlobalUtilityEnabledState
+- GP name: Configure global utility enabled state
+- GP path: Administrative Templates/Microsoft PowerToys
+- GP scope: Computer and user
+- ADMX file name: PowerToys.admx
+
+#### Registry information
+
+- Path: Software\Policies\PowerToys
+- Name: ConfigureGlobalUtilityEnabledState
+- Type: DWORD
+- Example value: `0x00000000`
+
+#### Intune information
+
+- OMA-URI: ./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys/ConfigureGlobalUtilityEnabledState
```

Comment:
The ~ might get interpreted. Needs to be improved.

Reply:
@alvinashcraft
Do you know if the `~` gets interpreted as stroken text? Found nothing on the syntax docs for MS Learn contribution.

Reply:
It looks correct in the VS Code preview pane with the Learn Authoring Pack plugins installed when copied this line into another file as a test. If it doesn't look right after publishing, we can try escaping the ~ chars.